server/server.js

```diff
@@ -16,10 +17,17 @@ app.use(express.static('client/dist'));
 
 
 
-
-app.get('/', (req, res) => {
-  res.status(200).send('Get request successful!');
-});
+app.get('/test', function (req, res) {
+  db.getAll((err, result) => {
+    if (err) {
+      res.send(`couldn't get`)
+      console.log(`we didn't get it`)
+    } else {
+      res.send(result)
+      console.log('we got it')
```

Comment:
Refactoring with Promises might be cleaner here.

Reply:
gonna stick with cb's till I get the hang of promises